spec/support/page_objects/publish/provider_courses_show.rb

```diff
@@ -24,6 +24,9 @@ class ProviderCoursesShow < PageObjects::Base
       section :other_requirements, Sections::SummaryList, '[data-qa="enrichment__other_requirements"]'
 
       element :basic_details_link, "a.govuk-link.govuk-tabs__tab", text: "Basic details"
+      element :status_sidebar, '[data-qa="course__status_panel"]'
+      element :unpublished_partial, '[data-qa="unpublished__partial"]'
+      element :published_partial, '[data-qa="published__partial"]'
```

Comment:
These should be nested in `status_sidebar`

Reply:
I think they can only be nested within a `section` ? Done